packages/discovery-provider/ddl/migrations/0074_usdc_purchase_splits_user_ids.sql

```diff
@@ -0,0 +1,175 @@
+BEGIN TRANSACTION;
+
+-- Update track stream price history
+WITH updated_track_stream_splits AS (
+    SELECT
+        track_id,
+        stream_conditions -> 'usdc_purchase' -> 'price' AS price,
+        jsonb_build_array(
+            jsonb_build_object('user_id', owner_id, 'percentage', 100.0000)
+        ) as splits
+    FROM
+        tracks
+    WHERE
+        jsonb_typeof(stream_conditions -> 'usdc_purchase' -> 'splits') = 'object'
+)
+UPDATE
+    track_price_history
+SET
+    splits = updated_track_stream_splits.splits
+FROM
+    updated_track_stream_splits
+WHERE
+    updated_track_stream_splits.track_id = track_price_history.track_id
+    AND track_price_history.access = 'stream';
+
+-- Update track stream conditions
+WITH updated_track_stream_splits AS (
+    SELECT
+        track_id,
+        stream_conditions -> 'usdc_purchase' -> 'price' AS price,
+        jsonb_build_array(
+            jsonb_build_object('user_id', owner_id, 'percentage', 100.0000)
+        ) as splits
+    FROM
+        tracks
+    WHERE
+        jsonb_typeof(stream_conditions -> 'usdc_purchase' -> 'splits') = 'object'
```

Comment:
do we have non object things still?

Reply:
this is really just `IS NOT NULL` i guess, but with a tighter safeguard for _after_ this conversion happens to not do it again